x-pack/plugins/session_view/common/utils/sort_processes.ts

```diff
@@ -8,15 +8,17 @@
 import { Process } from '../types/process_tree';
 
 export const sortProcesses = (a: Process, b: Process) => {
-  const eventAStartTime = new Date(a.getDetails().process.start);
-  const eventBStartTime = new Date(b.getDetails().process.start);
+  const eventAStartTime = a.getDetails()?.process.start;
+  const eventBStartTime = b.getDetails()?.process.start;
 
-  if (eventAStartTime < eventBStartTime) {
-    return -1;
-  }
+  if (eventAStartTime && eventBStartTime) {
```

Comment:
What if we set an equal value for `eventAStartTime` and `eventBStartTime` to avoid checking if either of them exists as well as the nested if